c_src/cb.c

```diff
@@ -651,6 +655,8 @@ ERL_NIF_TERM cb_http(ErlNifEnv* env, handle_t* handle, void* obj)
     ErlNifBinary value_binary;
     enif_alloc_binary(cb.ret.size, &value_binary);
     memcpy(value_binary.data, cb.ret.data, cb.ret.size);
+    free(cb.ret.data);
+    free(cb.ret.key);
```

Comment:
If I'm not mistaken, `data` and `key` are allocated by callbacks in callbacks.c for some functions. https://github.com/chitika/cberl/blob/master/c_src/callbacks.c#L111

`data` is allocated but `key` is not, in this case.